### PR TITLE
Updated MoEngage SDK dependency to 9.x.x

### PR DESCRIPTION
**Pull request by RakshithaAcharya**

### CODE REVIEW

**Review by umangmoe — APPROVED** [review]

File: MoEngagePluginBase.podspec [hunk 1]
```diff
@@ -22,6 +22,6 @@ Pod::Spec.new do |s|
     s.source_files = 'MoEngagePluginBase/**/*'
     s.swift_version = '5.0'
     s.frameworks = 'UIKit', 'Foundation', 'UserNotifications'
-    s.dependency 'MoEngage-iOS-SDK', '>= 8.4.0', '< 8.5.0'
-    s.dependency 'MoEngageInApp', '>= 3.4.0', '< 3.5.0'
+    s.dependency 'MoEngage-iOS-SDK', '>= 9.1.0', '< 9.2.0'
```
Comment (deepa-iOS-SDK) [inline MoEngagePluginBase.podspec:25]: @RakshithaAcharya As we have release 9.2.x should we updating it to the latest version?

Comment (RakshithaAcharya) [reply MoEngagePluginBase.podspec:25]: @deepa-iOS-SDK  ill update in main branch.. when I had raised 9.2.0 was not yet released

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 =================================
 
 ## Change Log:
+### Version 4.0.0   *(Date)*
```
Comment (deepa-iOS-SDK) [inline CHANGELOG.md:5]: @RakshithaAcharya Can you update the date here?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
...
+-------------------------------------------
+* MoEngage-iOS-SDK dependency updated to 9.1.* and MoEngageInApp dependency updated to 4.1.*.
```
Comment (deepa-iOS-SDK) [inline CHANGELOG.md:7]: @RakshithaAcharya Should be 9.2.x I believe

Comment (RakshithaAcharya) [reply CHANGELOG.md:7]: @deepa-iOS-SDK  ill do it main branch

**Review by deepa-iOS-SDK — APPROVED** [review]

**Pull request merged**